react/features/base/conference/middleware.js

```diff
@@ -255,6 +255,9 @@ function _setAudioOnly({ dispatch, getState }, next, action) {
     const result = next(action);
     const { audioOnly: newValue } = getState()['features/base/conference'];
 
+    // ensure to create video track when audioOnly mode is off
+    const ensureTrack = action.ensureVideoTrack && !newValue;
```

Comment:
No need to do this, setVideoMuted is smart enough to not create a track if muted is true.